scripts/skills/perks/perk_rf_vigorous_assault.nut

```diff
@@ -100,15 +100,18 @@ this.perk_rf_vigorous_assault <- ::inherit("scripts/skills/skill", {
 
 	function onAfterUpdate( _properties )
 	{
+		local actor = this.getContainer().getActor();
+		if (actor.isPreviewing() && actor.getPreviewSkill() != null)
+			return;
```

Comment:
Please add a comment to why this function ends early on previewing a skill. 
I assume that is because of the built-in condition of the perk whereas the bonus is lost when using any skill.